doc/subsystems/bluetooth/qualification.rst

```diff
@@ -15,3 +15,28 @@ the following documents:
    l2cap-pics.rst
    sm-pics.rst
    rfcomm-pics.rst
+
+Qualification Listings
+**********************
+
+The Zephyr BLE stack has obtained qualification listings for both the Host
+and the Controller.
+See the tables below for a list of qualification listings
+
+Host qualifications
+-------------------
+
+=============== ======================================================= ===================
+Zephyr version  Link                                                    Qualifying Company
+=============== ======================================================= ===================
+1.13            https://launchstudio.bluetooth.com/ListingDetails/70189 Nordic
+=============== ======================================================= ===================
+
+Controller qualifications
+-------------------------
+
+=============== ======================================================= =================== ===================
```

Comment:
Same here:

    .. list-table::
       :header-rows: 1

       * - Zephyr version
         - Link
         - Qualifying Company
         - Compatible Hardware

       * - 1.9 to 1.13
         - `Launch Studio details <https://launchstudio.bluetooth.com/ListingDetails/25166>`__
         - Nordic
         - nRF52x

Reply:
will switch, thanks!